ha/core/event_manager/model/action_event.py

```diff
@@ -29,20 +28,19 @@ def __init__(self, healthevent: HealthEvent):
         """
         Init method.
         """
-        self.event = Event()
         payload = {
-            HEALTH_ATTRIBUTES.SOURCE.value : healthevent.source,
-            HEALTH_ATTRIBUTES.CLUSTER_ID.value : healthevent.cluster_id,
-            HEALTH_ATTRIBUTES.SITE_ID.value : healthevent.site_id,
-            HEALTH_ATTRIBUTES.RACK_ID.value : healthevent.rack_id,
-            HEALTH_ATTRIBUTES.STORAGESET_ID.value : healthevent.storageset_id,
-            HEALTH_ATTRIBUTES.NODE_ID.value : healthevent.node_id,
-            HEALTH_ATTRIBUTES.RESOURCE_TYPE.value : healthevent.resource_type,
-            HEALTH_ATTRIBUTES.RESOURCE_ID.value : healthevent.resource_id,
-            HEALTH_ATTRIBUTES.RESOURCE_STATUS.value : healthevent.event_type,
-            HEALTH_ATTRIBUTES.SPECIFIC_INFO.value : healthevent.specific_info,
+            HealthAttr.SOURCE.value: healthevent.source,
+            HealthAttr.CLUSTER_ID.value: healthevent.cluster_id,
+            HealthAttr.SITE_ID.value: healthevent.site_id,
+            HealthAttr.RACK_ID.value: healthevent.rack_id,
+            HealthAttr.STORAGESET_ID.value: healthevent.storageset_id,
+            HealthAttr.NODE_ID.value: healthevent.node_id,
+            HealthAttr.RESOURCE_TYPE.value: healthevent.resource_type,
+            HealthAttr.RESOURCE_ID.value: healthevent.resource_id,
+            HealthAttr.RESOURCE_STATUS.value: healthevent.event_type
         }
-        self.event.set_payload(payload)
+        self.event = HEvent(**payload)
+        self.event.set_specific_info({HealthAttr.SPECIFIC_INFO.value: healthevent.specific_info})
```

Comment:
I guess we do not need to add the key: HealthAttr.SPECIFIC_INFO.value here. Please check.

Reply:
Done

Reply:
I see it is not removed yet @gauri-bhosale

Reply:
Fixed.